### DISCUSSION TITLE
Add documentation explaining how to integrate a password manager

### DISCUSSION THREAD

**Discussion by mthurmond** [minbrowser/min#2139] [Ideas]
The readme mentions "Password manager integration" but I can't find documentation on the suggested approach. I dug through a long github issue thread that seemed to imply this functionality is supported but parsing through it to figure out the supported path seems like it would take a while. 

I recommend having a suggested approach with step by step instructions in the documentation, as password auto-fill will be critical for most new users (lack of this is the main reason I don't use min as my primary browser). Details on how to migrate passwords from Chrome would also be helpful. 

I'm happy to write this documentation up if 1) someone can point me to the github issue(s) to read through, and 2) it will get added into the readme/docs.

Thanks!
Matt

**Comment by ghost** [comment]
Hi mthurmond, thank you for feedback.

> The readme mentions "Password manager integration" but I can't find documentation on the suggested approach. I dug through a long github issue thread that seemed to imply this functionality is supported but parsing through it to figure out the supported path seems like it would take a while.

So... I was thinking about [developing keepass for the min browser](https://github.com/minbrowser/min/discussions/2088), but I didn't find any tutorial or information how to do that.

**Comment by PalmerAL** [comment]
There is some existing integration support; you can enable it from settings > password auto-fill. It works by connecting to the CLI tools of Bitwarden/1password, or there's a basic built-in manager as well. Once you pick a manager, there should be a setup dialog that guides you through installing the CLI (or in the case of the built-in manager, it should just work by default).

Importing from Chrome isn't supported.

If you want to add support for a new manager, the thing to do would be to implement a class for it following the Bitwarden example: https://github.com/minbrowser/min/blob/master/js/passwordManager/bitwarden.js. For each supported command, you would need to make a call into the password manager's CLI tool and return the appropriate result.

**Reply by ghost** [reply]
Hi PalmerAL, thank you for feedback.

**Reply by mthurmond** [reply]
Thanks @PalmerAL and @codehangen for your feedback!

**Reply by ghost** [reply]
Hi mthurmond.

> Thanks @PalmerAL and @codehangen for your feedback!

Thanks for the comment here.
